### PR TITLE
Adding a static homepage to our website

### LINKED ISSUE #12: Adding a web page
Adding a static homepage to our website to replace the default Django webpage.

- Bootstrap our app.
- Register our app with Django.
- Creating the view function.
- Creating the template.
- Creating the route.

This issue is dependent on #9

### PR DESCRIPTION
**Pull request by oriAdler**
fix #12 
Adding a static homepage to our website:

    - Create a new app directory called 'homepage'.
    - Modify the 'setting.py' file to register our app with Django.
    - Create a new 'home' function in the 'views.py' file to render the new web page.
    - Modify the 'urls.py' file to create the route for the new web page.

### CODE REVIEW

File: homepage/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (lmilbaum) [inline homepage/admin.py]: Is this comment necessary?

File: homepage/models.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```
Comment (lmilbaum) [inline homepage/models.py]: same question

File: homepage/tests.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (lmilbaum) [inline homepage/tests.py]: same question

File: homepage/views.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+from django.shortcuts import render
+
+def home(request):
+    return render(request, 'homepage/home.html')
+
+# Create your views here.
```
Comment (lmilbaum) [inline homepage/views.py]: same question

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: homepage/admin.py (hunk 1 above)
Comment (oriAdler) [reply homepage/admin.py]: Absolutely not, I'll remove all comments.

**Review by lmilbaum — APPROVED** [review]

**Pull request merged**